Dockerfile

```diff
@@ -0,0 +1,136 @@
+FROM ubuntu:xenial
+MAINTAINER Thomas B. Mooney <tmooney@genome.wustl.edu>
+
+LABEL \
+    description="Image for tools used in alignment"
+
+RUN apt-get update -y && apt-get install -y \
+    ant \
+    apt-utils \
+    build-essential \
+    bzip2 \
+    default-jdk \
+    default-jre \
+    gcc-multilib \
+    git \
+    libncurses5-dev \
+    libnss-sss \
+    nodejs \
+    python-dev \
+    python-pip \
+    tzdata \
+    unzip \
+    wget \
+    zlib1g-dev
+
+##############
+#Picard 2.9.0#
```

Comment:
They are using 2.4.1. I'd copied this one from the rnaseq image I made, which used the latest versions of everything.